include/mega/megaclient.h

```diff
@@ -178,6 +181,31 @@ class MEGA_API FetchNodesStats
     dstime timeToTransfersResumed;
 };
 
+
+// Helper class for MegaClient.  Executes independent tasks on a separate thread or threads.
```

Comment:
Can the doc mention that this is a thread pool?

Reply:
done